docs/source/architecture/slashing.rst

```diff
@@ -0,0 +1,72 @@
+The Slashing Protocol
+=====================
+
+TBD
+
+
+Violations
+----------
+
+TBD
+
+
+Calculating the slashing penalty
+--------------------------------
+
+TBD (https://github.com/nucypher/nucypher/issues/803)
+
+
+How slashing affects stake
+--------------------------
+
+The goal of slashing is to reduce the number of tokens that belongs to a staking offender.
+In this case, the main task is not to violate the logic of locking tokens.
+The entire stake consists of:
+
+* tokens which the staker can withdraw at any moment
+* tokens locked for a specific period
+
+Since a staker can divide a stake (extend a part of a stake), as well as lock new tokens, then the total stake is represented as the sum of all the sub stakes active at a particular period. Each sub stake has a beginning and duration, which is reduced only upon confirmation of the activity. Sub stakes get slashed in order based on their duration, beginning with the shortest. Therefore, to start, the unlocked part of tokens belonging to the staker is slashed. After that, if necessary, sub stakes are decreased based on duration: the shortest sub stake is decreased by the required amount; if the adjustment of the sub stake is not enough, then the next shortest sub stake is decreased, and so on. A sub stake that begins in the next period is checked separately.
+
+**Example:**
+
+A staker has 1000 tokens:
+	* 1st sub stake = 500 tokens locked for 10 periods
+	* 2nd sub stake = 200 tokens for 2 periods
+	* 3rd sub stake = 100 tokens locked starting from the next period
+	* Remaining 200 tokens
+
+Penalty Scenarios:
+
+* *100 tokens*:
```

Comment:
Perhaps we can do something here for the scenario to stand out a bit better.

Maybe:
```suggestion
* *SCENARIO 1: 100 tokens*:
```
or something like that, and similar for the others (SCENARIO 2: 400 tokens, etc.). The bullet points for each scenario don't stand out enough in my mind. Thoughts?

Reply:
@derekpierre Definitely. Going further, I'd remind the reader that a punishment is due. 
`Scenario 1: Staker incurs penalty calculated to be worth *100 tokens*:`